### ISSUE TITLE
Filter links by date

### ISSUE THREAD

**Issue by ArthurHoaro** [shaarli/Shaarli#246]
This has been discussed in #106 by @nodiscc @virtualtam and @mrjovanovic.

This is my understanding:

  1. First step, add a simple search filter with two $_GET parameters (links between 2 dates).
  2. Add a datepicker as a plugin to handle these parameters. UI idea:

![img](https://cloud.githubusercontent.com/assets/4664783/6588359/940eca72-c75e-11e4-843a-99f03cc937da.png)

**Comment by nodiscc** [comment]
I agree except for the `Type date range here`  field placement. The search bar should only have text/tags fields, for simplicity (search by date is kind of an advanced search feature). Agreed with everything else.

**Comment by nicolasdanelon** [comment]
can it be added as a plugin?

**Comment by nodiscc** [comment]
> can it be added as a plugin?

There is no plugin system.

**Comment by nodiscc** [comment]
By the way, work on this can already be started, even if no UI implementation has been decided. Date filtering should work with URL parameters in the first time.

**Comment by nicolasdanelon** [comment]
Yes I know that there is no plugin system. I don't see this usefull, because of that I would like this in a plugin :)

**Comment by tobru** [comment]
I would _love_ too see this feature in Shaarli. This could help me to answer the question "which bookmarks have I added during the last week?".

My use case: I'd like to create a curated link-dump blogpost once in a while and it would help a lot if I could filter bookmarks using a date-range filter. F.e. "give me all links with the tag linkdump which where added in the last two weeks".
